web/html/src/components/table/SelectedRowDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+
+type SearchPanelPropss = {
+  /** number representing the number of the first displayed item */
+  fromItem: number;
+
+  /** number representing the number of the last displayed item */
+  toItem: number;
+
+  /** total number of filtered items */
+  itemCount: number;
+
+  /** number of selected items */
+  selectedCount: number;
+
+  /** flag indicating whether to show the number of selected items */
+  selectable?: boolean;
+
+  /** function called when the clear button is clicked. This should reset the selection. */
+  onClear: () => void;
+
+  /** function called when the Select All button is clicked. Should set the selection */
+  onSelectAll: () => void;
+};
+
+/** Panel containing the search fields for a table */
+export function SelectedRowDetails(props: SearchPanelPropss) {
+  return (
+    <>
+      {props.selectable && props.selectedCount > 0 && (
+        <div className="selected-row-details">
+          <span>
+            {props.selectedCount === props.itemCount ? (
+              <>
+                {t("All {totalCount} items are selected.", { totalCount: props.itemCount })}&nbsp;
+                <button className="btn btn-tertiary" onClick={props.onClear}>{t("Clear")}</button>
+              </>
+            ) : (
+              <>
+                {/* {t("{selectedCount} items selected.", { selectedCount: props.selectedCount })}&nbsp; */}
+                {props.selectedCount === 1
+                  ? t("1 item selected.")
+                  : t("{selectedCount} items selected.", { selectedCount: props.selectedCount })}
```

Comment:
Unfortunately this won't work for all languages, some languages have more cases than simply one or many. The correct way to localize this is to use [`{plural}`](https://formatjs.github.io/docs/core-concepts/icu-syntax/#plural-format).

```
{itemCount, plural,
  =0 {You have no items.}
  one {You have {itemCount, number} item.}
  other {You have {itemCount, number} items.}
}
```